Files jaxws-si.vanilla/.build.xml.swp and jaxws-si/.build.xml.swp differ
diff -Nru jaxws-si.vanilla/build.xml jaxws-si/build.xml
--- jaxws-si.vanilla/build.xml	2007-04-16 20:09:54.000000000 +0200
+++ jaxws-si/build.xml	2007-04-16 20:18:16.000000000 +0200
@@ -78,16 +78,10 @@
     <javac srcdir="src/rt/src" destdir="rt_build" debug="true">
       <classpath refid="runtime.compile.classpath"/>
     </javac>
+    <jar destfile="jax-ws.jar">
+      <fileset dir="rt_build" includes="**/*.*"/>
+    </jar>
 
-    <copy todir="tools_build">
-      <fileset dir="src/tools/wscompile/src">
-        <include name="**/*.properties" />
-      </fileset>
-    </copy>
-    <versionProp file="tools_build/com/sun/tools/ws/version.properties" />
-    <javac srcdir="src/tools/wscompile/src" destdir="tools_build" debug="true">
-      <classpath refid="tools.compile.classpath"/>
-    </javac>
   </target>
 
 </project>
Files jaxws-si.vanilla/src/rt/src/com/sun/xml/ws/message/stream/.StreamMessage.java.swp and jaxws-si/src/rt/src/com/sun/xml/ws/message/stream/.StreamMessage.java.swp differ
diff -Nru jaxws-si.vanilla/src/rt/src/com/sun/xml/ws/message/stream/StreamHeader.java jaxws-si/src/rt/src/com/sun/xml/ws/message/stream/StreamHeader.java
--- jaxws-si.vanilla/src/rt/src/com/sun/xml/ws/message/stream/StreamHeader.java	2007-04-16 20:09:54.000000000 +0200
+++ jaxws-si/src/rt/src/com/sun/xml/ws/message/stream/StreamHeader.java	2007-04-16 20:12:34.000000000 +0200
@@ -214,12 +214,8 @@
     }
 
     public void writeTo(XMLStreamWriter w) throws XMLStreamException {
-        try {
             // TODO what about in-scope namespaces
             _mark.writeToXMLStreamWriter(w);
-        } catch (XMLStreamBufferException e) {
-            throw new XMLStreamException2(e);
-        }
     }
 
     public void writeTo(SOAPMessage saaj) throws SOAPException {
diff -Nru jaxws-si.vanilla/src/rt/src/com/sun/xml/ws/message/stream/StreamMessage.java jaxws-si/src/rt/src/com/sun/xml/ws/message/stream/StreamMessage.java
--- jaxws-si.vanilla/src/rt/src/com/sun/xml/ws/message/stream/StreamMessage.java	2007-04-16 20:09:54.000000000 +0200
+++ jaxws-si/src/rt/src/com/sun/xml/ws/message/stream/StreamMessage.java	2007-04-16 20:13:34.000000000 +0200
@@ -287,8 +287,6 @@
             return new StreamMessage(envelopeTag, headerTag, attachmentSet, HeaderList.copy(headers), bodyTag, clone, soapVersion);
         } catch (XMLStreamException e) {
             throw new WebServiceException("Failed to copy a message",e);
-        } catch (XMLStreamBufferException e) {
-            throw new WebServiceException("Failed to copy a message",e);
         }
     }
 
